superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -125,6 +125,10 @@ const StyledToolbar = styled.div`
       }
     }
   }
+
+  .limitDropdown {
+    width: max-content;
```

Comment:
this is great! Super small nit, but `white-space: nowrap` may be more explicit here. You're saying "don't wrap on the white space", rather than "make the width of the element match the content." With `white-space`if you wanted to change the width later and make it larger, you still can.

Reply:
That's a lot safer of a method, thanks! Fixed in [`this commit`](https://github.com/apache/superset/pull/17382/commits/f7e464303fab2f3e52ad133bf087a6c3c01c5e0f).